shared/lib/detected_variables.py

```diff
@@ -109,10 +113,11 @@ def var_detection_result_to_dict(res: VarDetectionResult) -> Dict:
   return result
 
 
-def dict_to_var_detection_result(input: Dict) -> VarDetectionResult:
-  return VarDetectionResult(single_var=dict_to_var_candidates(input),
-                            multi_var=dict_to_multivar_candidates(
-                                input.get('MultiSV', {})))
+def test_dict_to_var_detection_result(input: Dict) -> VarDetectionResult:
```

Comment:
is this only used in testing (because of the 'test' prefix)? if so, add a comment

Reply:
Done.